package.json

```diff
@@ -22,9 +22,7 @@
         "echarts": "^5.4.1",
         "flatpickr": "^4.6.13",
         "laravel-vite-plugin": "^0.7.3",
-        "lodash": "^4.17.21",
         "luxon": "^3.2.1",
-        "postcss": "^8.4.21",
```

Comment:
Why these packages are removed from package.json? I'm pretty sure we use them in the project.

Reply:
I removed it accordingly to the upgrade guide recommendations: https://github.com/laravel/laravel/compare/9.x...10.x They are still in node modules as they're required by other packages.

Anyway, I'm returning them back as I cna see that lodash is used directly in some files.